db/migrate/20171027101922_create_meals.rb

```diff
@@ -0,0 +1,12 @@
+class CreateMeals < ActiveRecord::Migration[5.1]
+  def change
+    create_table :meals do |t|
+      t.string :name
+      t.string :course
+      t.integer :price
```

Comment:
I prefer to apply table alignment to migrations as well but it's not a rule so it's up to you.